bookkeeper-server/src/test/java/org/apache/bookkeeper/meta/GcLedgersTest.java

```diff
@@ -431,4 +657,59 @@ public boolean waitForLastAddConfirmedUpdate(long ledgerId,
             return false;
         }
     }
+
+    class LedgerManagerWrapper implements LedgerManager {
```

Comment:
I think you can use `CleanupLedgerManager` for this. You don't need to recreate a similar class here.

Reply:
ok